test/fixtures/list-item/0.md

```diff
@@ -2,7 +2,12 @@ All list-items in this document should be linted as **valid**.
 
 - [foo](https://foo.com) - Valid description.
 - [foo](https://foo.com) - A valid description.
+- [foo](https://foo.com) - A valid description...
 - [foo](https://foo.com) - A valid description!
+- [foo](https://foo.com) - A valid description!!!
+- [foo](https://foo.com) - A valid description?
+- [foo](https://foo.com) - A valid description???
+- [foo](https://foo.com) - A valid description????!??
```

Comment:
I don't think multiple `?` or `!` should be allowed, but not very important to prevent that though.

Reply:
Agreed